### PR TITLE
BI-10291 Redirecting to psc-statement page if company has no PSCs

### PR DESCRIPTION
**Pull request by SVarallo**

### CODE REVIEW

File: src/controllers/tasks/active.psc.details.controller.ts [hunk 1]
```diff
@@ -14,13 +14,18 @@ import {
...
 import { toReadableFormat } from "../../utils/date";
+import { logger } from "../../utils/logger";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
     const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
     const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
     const pscs: PersonOfSignificantControl[] = await getPscs(req.session as Session, transactionId, submissionId);
-
+    if (pscs.length < 1) {
```
Comment (markpit) [inline src/controllers/tasks/active.psc.details.controller.ts:24]: pscs might be undefined as well, i would include that in the if check like
if (!pscs || pscs.length < 1)

File: test/controllers/tasks/active.psc.details.controller.unit.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
+import { urlUtils } from "../../../src/utils/url";
```
Comment (markpit) [inline test/controllers/tasks/active.psc.details.controller.unit.ts:1]: move the urlUtils import so it is with the other imports, but make sure that the import mocks is still the first import, safer to add to bottom of imports.

File: test/controllers/tasks/active.psc.details.controller.unit.ts [hunk 3]
```diff
@@ -67,5 +70,16 @@ describe("Active psc details controller tests", () => {
       expect(response.text).toContain("Charity - Unincorporated Association");
       expect(response.text).toContain("Ownership of voting rights - more than 75%");
     });
+
+    it("should navigate to psc statement page if no psc is found", async () => {
```
Comment (markpit) [inline test/controllers/tasks/active.psc.details.controller.unit.ts:74]: also do a similar test that returns undefined from mockGetPscs.mockResolvedValueOnce
to test that it goes to the psc statement page as well (same as the test when [] gets returned)

File: test/controllers/tasks/active.psc.details.controller.unit.ts (hunk 2 above)
Comment (SVarallo) [reply test/controllers/tasks/active.psc.details.controller.unit.ts:1]: Ah my bad, it did that automatically

**Review by markpit — APPROVED** [review]

**Pull request merged**